### DISCUSSION TITLE
Can't apply for the student pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#54170] [GitHub Education]
I'm trying to apply for the Github Student Developer Pack, I can't upload proof of my academic status because of following reason: 
- The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
> My ID shows school name on the stamp, I even included a hand written note beside of it. 

I tried what i could, but it keeps rejecting my proof.

**Comment by hrishikesh-kocharekar-dev** [comment]
Make sure the image you're uploading is clear and legible. If the image is blurry or difficult to read, GitHub's verification system may not be able to recognize the school name.

Try taking a photo of your ID and your handwritten note separately, and then upload both images. This will give the verification team a clearer view of both documents.

If your ID doesn't have the school name stamped on it, you may need to provide additional documentation, such as a transcript or enrollment verification letter. You can check the eligibility requirements for the Student Developer Pack on the GitHub Education website to see what types of documents are accepted.

If you're still having trouble, you can reach out to GitHub's support team for assistance. They may be able to provide additional guidance or help you troubleshoot any issues with your submission. I hope these suggestions are helpful, and that you're able to successfully apply for the Student Developer Pack soon!

**Reply by nomikugg** [reply]
Como me pongo en contacto con soporte, no puedo enviar un correo al de soporte
